2023-02-14 01:13:33,713 INFO    MainThread:13055 [internal.py:wandb_internal():89] W&B internal server running at pid: 13055, started at: 2023-02-14 01:13:33.712208
2023-02-14 01:13:33,717 INFO    WriterThread:13055 [datastore.py:open_for_write():77] open: /home/bhavyajeet.singh/msme/MEMS-XF2T/generation/mT5-baseline/wandb/offline-run-20230214_011331-3saahrny/run-3saahrny.wandb
2023-02-14 01:13:33,721 DEBUG   HandlerThread:13055 [handler.py:handle_request():126] handle_request: run_start
2023-02-14 01:13:38,362 DEBUG   HandlerThread:13055 [meta.py:__init__():40] meta init
2023-02-14 01:13:38,362 DEBUG   HandlerThread:13055 [meta.py:__init__():54] meta init done
2023-02-14 01:13:38,362 DEBUG   HandlerThread:13055 [meta.py:probe():211] probe
2023-02-14 01:13:38,371 DEBUG   HandlerThread:13055 [meta.py:_setup_git():201] setup git
2023-02-14 01:13:38,426 DEBUG   HandlerThread:13055 [meta.py:_setup_git():208] setup git done
2023-02-14 01:13:38,426 DEBUG   HandlerThread:13055 [meta.py:_save_pip():58] save pip
2023-02-14 01:13:38,428 DEBUG   HandlerThread:13055 [meta.py:_save_pip():72] save pip done
2023-02-14 01:13:38,429 DEBUG   HandlerThread:13055 [meta.py:_save_conda():79] save conda
2023-02-14 01:13:51,510 DEBUG   HandlerThread:13055 [meta.py:_save_conda():87] save conda done
2023-02-14 01:13:51,510 DEBUG   HandlerThread:13055 [meta.py:probe():249] probe done
2023-02-14 01:14:04,762 WARNING MainThread:13055 [internal.py:wandb_internal():145] Internal process interrupt: 1
2023-02-14 01:14:05,689 WARNING MainThread:13055 [internal.py:wandb_internal():145] Internal process interrupt: 2
2023-02-14 01:14:05,689 ERROR   MainThread:13055 [internal.py:wandb_internal():148] Internal process interrupted.
2023-02-14 01:14:05,743 INFO    SenderThread:13055 [sender.py:finish():986] shutting down sender
2023-02-14 01:14:06,488 INFO    WriterThread:13055 [datastore.py:close():281] close: /home/bhavyajeet.singh/msme/MEMS-XF2T/generation/mT5-baseline/wandb/offline-run-20230214_011331-3saahrny/run-3saahrny.wandb
2023-02-14 01:14:06,496 INFO    HandlerThread:13055 [handler.py:finish():683] shutting down handler
2023-02-14 01:14:06,500 INFO    MainThread:13055 [internal.py:handle_exit():76] Internal process exited
